.github/workflows/osv-scanner.yml

```diff
@@ -0,0 +1,50 @@
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+name: OSV Scanner
+
+on:
+  schedule:
+    - cron: '0 0 1 * *'
+
+jobs:
+  run-osv-scanner:
+    name: Run OSV Scanner
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+      - name: Scan for vulnerabilities
+        run: |
+          docker run --rm -v $PWD:/repo -w /repo ghcr.io/google/osv-scanner:latest --format markdown -r . |  grep -vE '^Scanned.*packages$' > VULNERABILITY.md
```

Comment:
Why do you do this in the checkout step? You can simply create a new one and specify the docker image there

Reply:
we gonna need checkout step for the PR automation if not it will throw error on command `/usr/bin/git config --local --get remote.origin.url`. 
Further, if we use container of osv-scanner image, inside this container we need to manually install osv-scanner.

Reply:
Sure you need to do the checkout step, but you should do it as a separate step for the sake of clarity

Maybe I get you wrong, but why should I need to install `osv-scanner` when using `ghcr.io/google/osv-scanner:latest`?

Reply:
> why should I need to install osv-scanner when using ghcr.io/google/osv-scanner:latest?
Actually i was talking about being inside of an container and doing all steps inside it.
My bad, i get it wrong